docs/data-cookbooks/browse-generation.md

```diff
@@ -493,20 +498,20 @@ The provided example script used in the example goes through all granules and ad
 
 The processing lambda you construct will need to do the following:
 
-* Create a browse image file based on the input data, and stage it to a location accessible to both this task and the ```MoveGrranules``` task in a S3 bucket.
+* Create a browse image file based on the input data, and stage it to a location accessible to both this task and the ```FilesToGranules``` and ```MoveGranules``` tasks in a S3 bucket.
 * Add the browse file to the input granule files, making sure to set the granule fileType to ```browse```.
-* Update meta.input_granules with the updated granules list, as well as provide the files to be moved in the payload for ```MoveGranules``` as output from the task.
+* Update meta.input_granules with the updated granules list, as well as provide the files to be integrated by ```FilesToGranules``` as output from the task.
 
 
 ### Generating/updating CMR metadata
 
-If you do not already have a CMR file in the granules list, you will need to generate one for valid export.   This example's processing script generates and adds it to the ```MoveGranules``` file list via the payload  but it can be present in the InputGranules from the DiscoverGranules step as well if you'd prefer to pre-generate it.
+If you do not already have a CMR file in the granules list, you will need to generate one for valid export.   This example's processing script generates and adds it to the ```FilesToGranules``` file list via the payload  but it can be present in the InputGranules from the DiscoverGranules step as well if you'd prefer to pre-generate it.
 
-Both ```MoveGranules``` and ```CmrStep``` expect a valid CMR file to be available if you want to export to CMR.
+Both downstream steps ```MoveGranules``` and ```CmrStep``` expect a valid CMR file to be available if you want to export to CMR.
```

Comment:
Would prefer either of `MoveGranules and PostToCmr` or `MoveGranuleStep and CmrStep`.

Reply:
Fixed in 72b733b